frontend/src/metabase/dashboard/hooks/use-embed-theme.ts

```diff
@@ -7,9 +7,7 @@ import type { DisplayTheme } from "metabase/public/lib/types";
 
 import type { EmbedThemeControls } from "../types";
 
-export const useEmbedTheme = (
-  themeName: DisplayTheme = "light",
-): EmbedThemeControls => {
+export const useEmbedTheme = (): EmbedThemeControls => {
```

Comment:
`useEmbedTheme` has never been passed any values.

Reply:
thanks for catching that